iFixit/react-commerce
# Add product reviews

~~deploy_block   Blocked by #650~~ un_deploy_block 🟢 

This PR adds the product rating:

<img width="418" alt="Screenshot 2022-08-26 at 19 34 23" src="https://user-images.githubusercontent.com/4640135/186960668-7edace61-8c3a-4690-9065-23b25917fe6a.png">

And the product reviews section:

<img width="1093" alt="Screenshot 2022-08-26 at 19 34 56" src="https://user-images.githubusercontent.com/4640135/186960747-b07f19f8-8896-46e5-9dab-7eeedccdffd6.png">

## QA

1. Open Vercel preview
2. Go to [/Products/iphone-11-screen](https://react-commerce-3433jb7gp-ifixit.vercel.app/Products/iphone-11-screen)
3. Verify that you can see the product rating
4. Verify that you can see the product reviews section
5. Verify that "write a product review" buttons redirects to the users reviews (which is the current behavior in production, e.g. https://www.ifixit.com/Store/Mac/MacBook-Air-13-Inch-Late-2010-2017-Battery/IF188-113?o=2)
